bin/fixedcase/truelist

```diff
@@ -1,535 +1,3150 @@
+Al Baḩr al Aḩmar
+Al Baṭḩah
+Al Buhayrah
+Al Buḩayrah
+Al Buraymī
+Al Buţnān
+Al Bāhah
+Al Bāţinah
+Al Farwānīyah
+Al Fayyūm
+Al Fujayrah
+Al Gharbīyah
+Al Haouz
+Al Hasakah
+Al Hoceïma
+Al Iskandarīyah
+Al Ismā`īlīyah
+Al Jabal al Akhḑar
+Al Jabal al Gharbī
+Al Jahrrā’
+Al Janūbī
+Al Janūbīyah
+Al Jawf
+Al Jazīrah
+Al Jifārah
+Al Jufrah
+Al Jīzah
+Al Karak
+Al Kharţūm
+Al Khawr wa adh Dhakhīrah
+Al Kufrah
+Al Kuwayt
+Al Ladhiqiyah
+Al Madīnah
+Al Mafraq
+Al Mahrah
+Al Manāmah
+Al Marj
+Al Marqab
+Al Maḩwīt
+Al Minyā
+Al Minūfīyah
+Al Muthanna
+Al Muḩarraq
+Al Qadisiyah
+Al Qalyūbīyah
+Al Qaşīm
+Al Qaḑārif
+Al Qunaytirah
+Al Qāhirah
+Al Wakrah
+Al Wusţá
+Al Wādī al Jadīd
+Al Wāḩāt
+Al Ḥudūd ash Shamāliyah
+Al Ḩudaydah
+Al ‘Aqabah
 Alabama
+Alabat Island Agta
+Alacatlatzala Mixtec
+Alago
+Alagoas
+Alagwa
+Alajuela
+Alak
+Alamblak
+Alangan
+Alanic
+Alapmunte
 Alaska
+Alaska Inupiatun
+Alaskan Inupiatun
+Alawa
+Alba
+Albacete
+Albania
 Albanian
+Albanian Sign Language
+Albay
+Albay Bikol
 Alberta
+Alcozauca Mixtec
+Alege
+Alekano
 Alergian
+Alessandria
 Aleut
 Alex
 Alexa
 Alexa Prize
+Alger
+Algeria
 Algerian
+Algerian Sign Language
+Algiers
 Algonquian
 Algonquin
 Ali
+Ali Sabieh
 Alibaba
+Alibori
 Alicante
+Alifu Alifu
+Alifu Dhaalu
+Alladian
+Allar
 Allen
+Allier
+Almaty
+Almaty oblysy
+Almería
+Alngith
+Alo Phola
+Alojas novads
+Alor
+Aloápam Zapotec
+Alsea
+Alsungas novads
+Alta
+Alta Verapaz
+Altai
+Altay
+Altayskiy kray
+Alto Paraguay
+Alto Paraná
+Alu Kurumba
+Alugu
+Alune
+Aluo
+Alur
+Alutor
 Alvarez
+Alyawarr
+Alytaus Apskritis
 Alzheimer
+Alūksnes novads
+Ama
+Amahai
+Amahuaca
+Amaimon
+Amal
+Amambay
+Amanab
+Amanayé
+Amapá
+Amara
+Amarakaeri
 Amarakodsa
+Amarasi
 Amarna
+Amasya
+Amatas novads
+Amatlán Zapotec
 Amazon
 Amazon Mechanical Turk
 Amazon.com
+Amazonas
 Amazonian
+Amba
+Ambae
+Ambai
+Ambakich
+Ambala Ayta
+Ambelau
+Ambele
+Amblong
+Ambonese Malay
+Ambrak
+Ambrym
+Ambul
+Ambulas
 Amdang
+Amdo Tibetan
+Amele
+America
 American
+American Samoa
 American Sign Language
 Americans
 Americas
+Amganad Ifugao
 Amharic
-Amo
+Ami
+Amis
+Ammóchostos
+Amnat Charoen
 Amobee
 Amodei
+Amol
+Amolatar
+Amoltepec Mixtec
+Ampanang
+Ampari Dogon
+Ampāara
+Amri Karbi
 Amsterdam
+Amto
 Amun
+Amundava
+Amurdak
+Amuria
+Amurskaya oblast'
+Amuru
+An Giang
+An Najef
+An Nuqaţ al Khams
+An Nīl
+An Nīl al Abyaḑ
+An Nīl al Azraq
+Ana Tinga Dogon
+Anaang
+Anabar
+Anakalangu
 Analy
+Anam
+Anambra
+Anambé
+Anamgura
+Anasi
+Anatoliki Makedonia kai Thraki
+Ancash
+Ancona
+Andaandi
+Andai
+Andajin
+Andalucía
 Andalusian
+Andaman Creole Hindi
+Andaman and Nicobar Islands
+Andaqui
+Andarum
+Andegerebinha
 Anderson
+Andh
+Andhra Pradesh
+Andi
+Andijon
+Andio
+Andjazîdja
+Andjouân
 Ando
+Andoa
+Andoque
+Andorra
+Andorra la Vella
+Andrijevica
+Andros
 Andrus
+Aneityum
+Anem
+Aneme Wake
+Anenii Noi
+Anetan
 Anfillo
+Ang Thong
+Angaataha
+Angaité
+Angal
+Angal Enen
+Angal Heneng
+Angami Naga
+Angaur
+Angguruk Yali
 Angika
+Angkamuthi
+Anglesey; Sir Ynys Môn
```

Comment:
Semicolon

Reply:
Oops yeah. There appear to be 15 semicolons for locations that were for some reason given names in two versions. I'll split those

Reply:
Majority of these were Welsh cities that have an English and a Gaelic name